server/models/Expense.js

```diff
@@ -125,19 +142,22 @@ module.exports = function (Sequelize, DataTypes) {
     },
 
     instanceMethods: {
-      reject() {
+      setApproved() {
+        this.status = status.APPROVED;
+        return this.save();
+      },
+
+      setRejected() {
         this.status = status.REJECTED;
         return this.save();
       },
 
-      approve() {
-        this.status = status.APPROVED;
+      setPaid() {
+        this.status = status.PAID;
         return this.save();
       }
     }
   });
 
-  return Expense;
-  // TODO: enable Temporal
-  // return Temporal(Expense, Sequelize);
+  return Temporal(Expense, Sequelize);
```

Comment:
If I remember right, we'll also need to add a migration file for the new Temporal table.